libvast/vast/hash/xxhash.hpp

```diff
@@ -30,9 +30,14 @@ static constexpr int xxh_force_memory_access = XXH_FORCE_MEMORY_ACCESS;
 static constexpr int xxh_force_memory_access = 0;
 #endif
 
-/// Allow null pointer input when hashing data of length greater 0.
+/// Allow null pointer input when hashing data of length greater 0. Enabled by
+/// default for xxHash >= 8.0.1.
```

Comment:
```suggestion
/// Allow null pointer input when hashing data of length greater 0
/// Since xxHash 8.0.1, all functions operate on null pointers, as long as
/// the size parameter is 0.
```